src/Lecture3.hs

```diff
@@ -137,10 +154,16 @@ data List1 a = List1 a [a]
 
 -- | This should be list append.
 instance Semigroup (List1 a) where
-
+    (<>) :: List1 a -> List1 a -> List1 a
+    List1 l ls <> List1 m ms =
+        List1 l (ls ++ (m : ms))
 
 {- | Does 'List1' have the 'Monoid' instance? If no then why?
 
+It cannot, as there's no List1 element such that when combined with another
+List1 element returns the same element (i.e. at least one item will always
+be appended to the original list).
```

Comment:
That's correct! 👍🏻 
Or, in other words, there's no such thing as "empty non-empty list" 🙂